Sources/Metric Logs/Components/HistogramView.swift

```diff
@@ -0,0 +1,280 @@
+                                               using formatter: MeasurementFormatter, numberFormatter: NumberFormatter = .standard) -> some View {
+        return HistogramView<T>(title: title, histogram: histogram,
+                                measurementFormatter: formatter, numberFormatter: numberFormatter)
+            .navigationTitle(title)
+    }
+
+    // MARK: - Properties
+    let title: String
+    let histogram: Histogram<T>
+    let measurementFormatter: MeasurementFormatter
+    let numberFormatter: NumberFormatter
+
+    @State private var relativePanLocation: CGPoint?
+    @State private var currentlyPannedEntry: Histogram<T>.Bucket?
+    @State private var informationViewSize: CGSize = .zero
+
+    private var backgroundGray: Color { Color(UIColor.lightGray).opacity(0.5) }
+
+    // MARK: - View
+    var body: some View {
+        let maxValue = Double(histogram.buckets.map(\.count).max() ?? 0) * 1.1
+        let indexedEntries = Array(zip(histogram.buckets.indices, histogram.buckets))
+
+        VStack(alignment: .trailing, spacing: 0) {
+            if let highestFrequencyEntry = histogram.highestFrequencyEntry {
+                TitleView(title: highestFrequencyEntry.formatted(using: measurementFormatter),
+                          subtitle: "Highest Frequency \(title)")
+                 Spacer()
+            }
+
+            HStack(spacing: 0) {
+                AxisView.displaying(countRange: 0...maxValue, along: .vertical, using: numberFormatter)
+                    .foregroundColor(backgroundGray)
+
+                ZStack(alignment: .bottom) {
+                    VStack {
+                        GeometryReader { geometry in
+                            HStack(alignment: .bottom) {
+                                ForEach(indexedEntries, id: \.0) { index, entry in
+                                    ItemView(unitValue: entry.unitValue(relativeTo: maxValue),
+                                             absoluteValue: entry.count,
+                                             label: entry.formatted(using: measurementFormatter),
+                                             isCurrentlySelected: currentlyPannedEntry == entry,
+                                             inSelectedMode: currentlyPannedEntry != nil)
+                                }
+                            }
+                            .gesture(DragGesture(minimumDistance: 0)
+                                        .onChanged { self.handlePanChanged(to: $0.location,
+                                                                           in: geometry.frame(in: .local), isEnding: false) }
+                                        .onEnded { self.handlePanChanged(to: $0.location,
+                                                                         in: geometry.frame(in: .local), isEnding: true) })
+                        }
+                    }
+                    .padding([.leading, .trailing])
```

Comment:
horizontal combines leading and trailing
```suggestion
                    .padding(.horizontal)
```